src/units.jl

```diff
@@ -1,3 +1,23 @@
+#########
+## Move attributes into certain unit spaces
+# Future:
+# Every plot / scene axis will hold a space attribute
+# with the following allowed (per axis):
+# (may not actually be a type hierarchy, just here for illustration)
+abstract type DataSpace end
+abstract type DisplaySpace end
+#...
+struct Log <: DataSpace end
+struct Polar <: DataSpace end
```

Comment:
Most nonlinear projections will need their own axis types...this is something to discuss.